### PR TITLE
AMP-963 NER Output in CSV Format

### PR DESCRIPTION
**Pull request by shea12**
Creates a new tool that takes NER output, converts it to CSV, and writes out CSV file.

### CODE REVIEW

File: tools/amp_ner/ner_to_csv.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<tool id="ner_to_csv" name="NER to CSV" version="1.1.0">
+  <description>Convert NER output to CSV</description>
```
Comment (yingfeng-iu) [inline tools/amp_ner/ner_to_csv.xml:2]: In a similar style  as the VTT generator, we can have the name as "AMP Named Entity JSON to CSV Converter"

File: tools/amp_ner/ner_to_csv.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  </requirements>
+  <command detect_errors="exit_code">
+    '$__tool_directory__/ner_to_csv_.py' '$amp_entities' '$amp_entities_csv';
+  </command>
+  <inputs>
+    <param name="amp_entities" type="data" format="ner" label="AMP Named Entities" help="AMP NER output file" ></param>
+  </inputs>
+  <outputs>
+    <data name="amp_entities_csv" format="csv" label="AMP Entities Extracted CSV"/>
+  </outputs>
```
Comment (yingfeng-iu) [inline tools/amp_ner/ner_to_csv.xml:13]: How about have the label as "AMP Entities CSV Converted"

File: tools/amp_ner/ner_to_csv.xml (hunk 1 above)
Comment (shea12) [reply tools/amp_ner/ner_to_csv.xml:2]: Done

File: tools/amp_ner/ner_to_csv.xml (hunk 2 above)
Comment (shea12) [reply tools/amp_ner/ner_to_csv.xml:13]: And done!

**Review by yingfeng-iu — APPROVED** [review]
I made a few suggestions on naming and requirements., otherwise looks good!

File: tools/amp_ner/ner_to_csv.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<tool id="ner_to_csv" name="AMP Named Entity JSON to CSV Converter" version="1.1.0">
+  <description>Convert NER output to CSV</description>
+  <requirements>
+  </requirements>
```
Comment (yingfeng-iu) [inline tools/amp_ner/ner_to_csv.xml:4]: We should add python as one requirement, see hmgm_ner.xml for example.

In general (not the case here) if there're other library packages (those not included in standard python) we should put in the requirements with its versions too.

**Pull request merged**